src/components/BoardOptionsMenu/BoardOptionsMenu.tsx

```diff
@@ -223,7 +224,7 @@ const BoardOptionsMenu = () => {
             </div>
             <hr />
             <div className="flex flex-col">
-              <input type="file" id="file" ref={fileInput} onChange={(e) => handleImportAll(e)} className="hidden" />
+              <input type="file" id="file" data-testid="file" ref={fileInput} onChange={(e) => handleImportAll(e)} className="hidden" />
```

Comment:
for testing purposes, this attribute has been added to get the element in the test by this id